docs/extract-example-block-usage-from-flows.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * This script is designed to explore the configuration used for particular blocks in flows saved to Flow Cloud.
+ * It can optionally filter by a specific config property, allowing for more targeted analysis.
+ *
+ * Why It's Useful:
+ * - Helps identify patterns or common configurations for specific block types.
+ * - Assists in auditing and understanding how a particular property is being used across multiple flows.
+ *
+ * How to Use:
+ * 1. Modify the variables below to specify the block type and property of interest (if any).
+ * 2. Run the script using Node.js.
+ * 3. Check the generated JSON file for the grouped data.
+ */
+
+
+let inputFilePath = 'flows.json';
+let outputFilePath = 'flow-block-config-analysis.json';
```

Comment:
I don't find it clear where the files will be saved/outputted.
Also I don't think `root` is the best place. What about create a `flow-analysis` folder and keep these files there? 
And then we can hide from git so we don't deploy it.

Reply:
Done @gsambrotta